portfolio/src/main/java/com/google/sps/servlets/TestServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/introduction")
+public class TestServlet extends HttpServlet {
+
+  @Override
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
```

Comment:
I don't see much benefit marking the parameters as `final` since we won't mutate parameters for sure.

Is it possible to mark this class as `final` here? It's always recommended to mark new class as `final` so that inheritance can be prohibited where possible.